services/web-app/src/app/profile/page.tsx

```diff
@@ -0,0 +1,59 @@
+"use client";
+import Loading from "@/components/loading/loading";
+import { RepoTable } from "@/components/tables/repoTable";
+import { useSession } from "next-auth/react";
+import Image from "next/image";
+import Link from "next/link";
+
+export default function Profile(): JSX.Element {
+  const repos = [
+    "code-review-gpt",
+    "cooking-website",
+    "make-money-fast-crypto",
+  ];
+
+  const { data: session, status } = useSession();
+
+  if (status === "loading") {
+    return <Loading />;
+  }
+
+  if (!session || session === null) {
```

Comment:
you shouldnt need the session === null. null is a falsy value so this is included in !session